mitxpro/templates/base.html

```diff
@@ -1,82 +1,98 @@
-          media="none" onload="if(media!='all') media='all'">
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,600,700|Rajdhani:300,400,500,600,700&display=swap"
+      media="all"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons&display=swap"
+      media="none"
+      onload="if(media!='all') media='all'"
+    />
     {% js_settings %}
     <script type="text/javascript">
-    {% if CSOURCE_PAYLOAD %}
-    var CSOURCE_PAYLOAD = {{ CSOURCE_PAYLOAD|safe }};
-    {% else %}
-    var CSOURCE_PAYLOAD = null;
-    {% endif %}
+      {% if CSOURCE_PAYLOAD %}
+      var CSOURCE_PAYLOAD = {{ CSOURCE_PAYLOAD|safe }};
+      {% else %}
+      var CSOURCE_PAYLOAD = null;
+      {% endif %}
     </script>
     {% render_bundle 'style' %}
     {% render_bundle 'common' %}
     {% render_bundle 'header-root' %}
 
     <title>{% block title %}{% endblock %}</title>
-    <meta name="description" content="{% block description %}{% endblock %}">
-    <meta name="keywords" content="{% block keywords %}{% endblock %}">
+    <meta name="description" content="{% block description %}{% endblock %}" />
+    <meta name="keywords" content="{% block keywords %}{% endblock %}" />
     {% if domain_verification_tag %}
-      <meta name="google-site-verification" content="{{ domain_verification_tag }}" />
+    <meta
+      name="google-site-verification"
+      content="{{ domain_verification_tag }}"
+    />
     {% endif %}
-    <meta name='zd-site-verification' content="o0itxdfh369m1bw4cuz9" />
-    <meta name='zd-site-verification' content="7e22kzikbruclk2jjomono" />
+    <meta name="zd-site-verification" content="o0itxdfh369m1bw4cuz9" />
+    <meta name="zd-site-verification" content="7e22kzikbruclk2jjomono" />
     {% block extrahead %}
     {% endblock %}
     {% endspaceless %}
-      {% if zendesk_config.help_widget_enabled and not request.path|startswith:'/certificate/' %}
-          <script defer id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key={{zendesk_config.help_widget_key}}"> </script>
-      {% endif %}
+    {% if zendesk_config.help_widget_enabled and not request.path|startswith:'/certificate/' %}
+    <script
+      defer
+      id="ze-snippet"
+      src="https://static.zdassets.com/ekr/snippet.js?key={{zendesk_config.help_widget_key}}"
+    ></script>
+    {% endif %}
   </head>
   <body class="{% block bodyclass %}{% endblock %}">
-  <div class="main-panel">
-    {% include "partials/gtm_body.html" %}
-    {# Site-wide notifications should not show up on the ecommerce bulk pages #}
-    {% if not request.path|startswith:'/ecommerce/bulk/,/certificate/' %}
+    <div class="main-panel">
+      {% include "partials/gtm_body.html" %}
+      {# Site-wide notifications should not show up on the ecommerce bulk pages
+      #}
```

Comment:
This is simple html text now. It is displayed on the front end.

<img width="1748" alt="Screenshot 2024-05-03 at 4 52 52 PM" src="https://github.com/mitodl/mitxpro/assets/52656433/0fef9da9-d4f5-415b-b808-a8b88984380d">

Reply:
Thanks for being thorough. This is fixed now.